composer.json

```diff
@@ -23,20 +23,20 @@
 	"repositories": [
 		{
 			"type": "vcs",
-			"url": "https://github.com/tomzx/gitter"
+			"url": "https://github.com/mallardduck/gitter"
```

Comment:
Note: This would be reverted once this PR https://github.com/tomzx/gitter/pull/4/files is merged in.